skycoin/skywire
# Add stcpr heartbeat

Did you run `make format && make check`?
yes

Fixes #	

 Changes:	
- Added GET `heatrbeat/stcpr` endpoint 
- Removed extra dial logs
- Added GetPersistentTransports api

How to test this PR:
1. Start the visor with test config and address resolver as `http://address-resolver.myownsky.xyz`, `./skywire-visor skywire-config.json` on a machine with public IP.
2. Check `http://address-resolver.myownsky.xyz/transports` if pk shows up.
